etna/records/views/records.py

```diff
@@ -76,10 +83,28 @@ def record_detail_view(request, iaid):
     # if page.is_digitised:
     #     image = Image.search.filter(rid=page.media_reference_id).first()
 
+    # Back to search - default url
+    back_to_search_url = reverse("search-featured")
+
+    # Back to search button - update url when timestamp is not expired
+    if back_to_search_url_timestamp := request.session.get(
+        "back_to_search_url_timestamp"
+    ):
+        back_to_search_url_timestamp = datetime.datetime.strptime(
+            back_to_search_url_timestamp, "%Y%m%d-%H%M%S"
+        )
+        back_to_search_url_timestamp = back_to_search_url_timestamp.astimezone(pytz.utc)
```

Comment:
If we use `datetime.isoformat()` and `datetime.fromisoformat()`, we shouldn't have to worry about the timezone here

Reply:
Done.